### ISSUE TITLE
Error when answering the card in version 2.1.26

### ISSUE THREAD

**Issue by cjdduarte** [my-Anki/Beautify-Anki#5]
First of all, congratulations on your addon.
I saw that you quoted my addon. Thanks for the reference.

> The Statistics part in the Deck Browser is based on Carlos Duarte  addon  More Decks Stats and Time Left 

Come on:

```
Anki 2.1.26 (70784154) Python 3.8.0 Qt 5.14.1 PyQt 5.14.1
Platform: Windows 10
Flags: frz=True ao=True sv=1
Add-ons, last update check: 2020-05-19 08:37:52

Caught exception:
Traceback (most recent call last):
  File "aqt\webview.py", line 464, in handler
  File "aqt\reviewer.py", line 540, in _onTypedAnswer
  File "aqt\reviewer.py", line 243, in _showAnswer
  File "aqt\reviewer.py", line 598, in _showEaseButtons
  File "<D:\Anki\Anki.2.1.26\decorator.pyc:decorator-gen-24>", line 2, in _answerButtons
  File "lib\site-packages\anki\hooks.py", line 638, in decorator_wrapper
  File "lib\site-packages\anki\hooks.py", line 630, in repl
  File "D:\Anki\Anki.2.1.desenv\addons21\1150874988\reviewer.py", line 122, in answerButtons
    buf += but(ease, label)
  File "D:\Anki\Anki.2.1.desenv\addons21\1150874988\reviewer.py", line 112, in but
    color,
UnboundLocalError: local variable 'color' referenced before assignment

```

**Comment by ShoroukAziz** [comment]
I loved your addon and it was a great help so thanks for developing it in the first place.
As for the error, Disable this addon https://ankiweb.net/shared/info/2494384865.
Please let me know if that fixes the issue

**Comment by cjdduarte** [comment]
I disabled all addons and the error continues.

![image](https://user-images.githubusercontent.com/42243532/82376348-54b04a00-99f8-11ea-9f1a-f1bff8c4f6c1.png)

**Comment by cjdduarte** [comment]
I found that when I use Anki in "Brazilian Portuguese", the error occurs.
When I moved to "English (United States)", it worked.

![image](https://user-images.githubusercontent.com/42243532/82376632-c7b9c080-99f8-11ea-8bb8-2a4cb6124de6.png)

**Comment by ShoroukAziz** [comment]
Interesting discovery. I was able to reproduce the error. and now I know why it happened so it'll be fixed in the next update. Thanks a lot.

**Comment by addala2005** [comment]
Thank You very much. It worked for me also.

**Comment by ShoroukAziz** [comment]
Hi, This is now fixed 5ba4867 
You can either download the addon from Github now. or, wait for the official update on Ankiweb.
